src/components/ProductTours/ProductTours.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { ProductTour } from '@edx/paragon';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import Cookies from 'universal-cookie';
+
+import { ROUTE_NAMES } from '../EnterpriseApp/constants';
+import { SETTINGS_TABS_VALUES } from '../settings/data/constants';
+import { features } from '../../config';
+import {
+  BROWSE_AND_REQUEST_TOUR_COOKIE_NAME,
+  LEARNER_CREDIT_COOKIE_NAME,
+  TOUR_TARGETS,
+} from './constants';
+import BrowseAndRequestTour from './BrowseAndRequestTour';
+import LearnerCreditTour from './LearnerCreditTour';
+import { SubsidyRequestsContext } from '../subsidy-requests';
+
+const cookies = new Cookies();
+
+const ProductTours = ({ enterpriseSlug }) => {
+
+  // All the logic here is for determining what ProductTours we should show
+  // All actual tour specific logic should live within the separate tour files
+  
+
+  const history = useHistory();
+  const inSettingsPage = history.location.pathname.includes(ROUTE_NAMES.settings);
+
+  // Browse and Request "Show Tour?" Logic
+  const { subsidyRequestConfiguration } = useContext(SubsidyRequestsContext);
+  const isBrowseAndRequestEnabledForEnterprise = features.FEATURE_BROWSE_AND_REQUEST;
+
+  const dismissedBrowseAndRequestTourCookie = cookies.get(BROWSE_AND_REQUEST_TOUR_COOKIE_NAME);
+
+  // Only show tour if feature is enabled, the enterprise is eligible for the feature,
+  // hide cookie is undefined or false, not in settings page, and subsidy requests are not already enabled
+  const showBrowseAndRequestTour = isBrowseAndRequestEnabledForEnterprise
+    && !dismissedBrowseAndRequestTourCookie && !inSettingsPage && !subsidyRequestConfiguration?.subsidyRequestsEnabled;
+
+  const [browseAndRequestTourEnabled, setBrowseAndRequestTourEnabled] = useState(showBrowseAndRequestTour);
+
+
+  // Learner Credit Management "Show Tour?" Logic
+  const isLearnerCreditEnabledForEnterprise = features.FEATURE_LEARNER_CREDIT_MANAGEMENT;
+
+  const dismissedLearnerCreditTourCookie = cookies.get(LEARNER_CREDIT_COOKIE_NAME);
+
+  // Only show tour if feature is enabled, the enterprise is eligible for the feature,
+  // hide cookie is undefined or false, not in settings page // and learner credit is enabled
+  const showLearnerCreditTour = isLearnerCreditEnabledForEnterprise
+    && !dismissedLearnerCreditTourCookie && !inSettingsPage // && !subsidyRequestConfiguration?.subsidyRequestsEnabled;
+
+  const [learnerCreditEnabled, setLearnerCreditTourEnabled] = useState(showLearnerCreditTour);
+
+
+  let tours = []
```

Comment:
Rather than pushing to this array conditionally, where once a tour is added, it's implied to be `enabled` too, could it might make more sense to structure this something like:

```jsx
const tours = [
  learnerCreditTour({
    enabled: learnerCreditEnabled,
    ...
  }),
  browseAndRequestTour({
    enabled: browseAndRequestEnabled,
    ...
  }),
];
```

Reply:
addressed in incoming commit 👍